pkg/command/map_string.go

```diff
@@ -36,6 +36,18 @@ func GetStringMapStringE(vp *viper.Viper, key string) (map[string]string, error)
 	if data == nil {
```

Comment:
duplicated nil-check and returning empty map can be removed because this is already done in the invoked `ToStringMapStringE` function

-> only `return ToStringMapStringE(vp.Get(key))` in body